### DISCUSSION TITLE
Difference between "pages" and "components" in Quasar file structure

### DISCUSSION THREAD

**Discussion by sumant4ssm** [quasarframework/quasar#10483] [General - Components / Directives / etc]
Hello,

I have been using **components** mostly but with quasar I noticed there are some extra folder like **pages**, **boot** and so on, I am wondering whats the use of each folder in Quasar structure. Is there a video or article explaning the use of each folder ?

Thank You,
Sumant

**Comment by MilosPaunovic** [comment]
https://quasar.dev/quasar-cli/directory-structure

**Reply by wsw70** [reply]
The difference between `pages` and `layouts` is not obvious, though (at least for me)
